[vtk] Port vtkStringFormatter to fmt 12

The fmt::localtime function is deprecated in 11.2 and removed in fmt 12.

--- Common/Core/vtkStringFormatter.h
+++ Common/Core/vtkStringFormatter.h
@@ -245,7 +245,9 @@ using fmt::format_to_n;
  *
  * @note The standard library provides this function, but it's not thread-safe.
  */
+#if 0
 using fmt::localtime;
+#endif
 
 /**
  * Converts given time since epoch as ``std::time_t`` value into calendar time,
